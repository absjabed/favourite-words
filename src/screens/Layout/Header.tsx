import React from 'react';
import {StyleSheet, Text, View} from 'react-native';
const Header = () => {
  return (
    <View style={styles.main}>
      <View style={styles.container}>
            <Text style={styles.text}>Dictonary Word Finder</Text>
      </View>
    </View>
  );
};

export default Header;

const styles = StyleSheet.create({
  main: {
    width: '100%',
    justifyContent: 'center',
    padding: 10,
    paddingHorizontal: 25,
    borderBottomColor: '#eee',
    borderBottomWidth: 1,
    backgroundColor: '#f2f2f2',
  },
  container: {
    width: '100%',
    height: 35,
    maxHeight: 40,
    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center',
  },

  text: {
    flexDirection: 'row',
    fontSize: 16,
    fontFamily:'Ubuntu-Regular',
    justifyContent: 'center',
  },
});
