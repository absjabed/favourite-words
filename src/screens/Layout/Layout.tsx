import React from 'react';
import {StyleSheet, View, ViewProps} from 'react-native';

const Layout : React.FC<ViewProps> = ({children}) => {
  return <View style={styles.screen}>{children}</View>;
};

export default Layout;

const styles = StyleSheet.create({
  screen: {
    flex: 1,
    fontFamily: 'Ubuntu-Regular',
  },
});
