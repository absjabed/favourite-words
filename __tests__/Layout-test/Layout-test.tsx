import 'react-native';
import React from 'react';
import Layout from '../../src/screens/Layout/Layout';
import renderer, { create } from 'react-test-renderer';

it('Layout component renders correctly', () => {
  renderer.create(<Layout />);
});

test('Layout snapshot', () => {
  const tree = create(<Layout/>);
  expect(tree).toMatchSnapshot();
});
