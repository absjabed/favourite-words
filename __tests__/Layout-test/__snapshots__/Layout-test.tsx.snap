// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "fontFamily": "Ubuntu-Regular",
    }
  }
/>
`;
